src/screens/Home/HomeUtils.res

```diff
@@ -215,7 +215,7 @@ module ControlCenter = {
           />
           <img
             alt="connector-list"
-            className="inline-block absolute right-5 bottom-5  lg:block"
+            className="opacity-10 sm:opacity-100 inline-block absolute right-5 bottom-5  lg:block"
```

Comment:
Using opacity to control responsiveness might not be ideal here. Consider using width classes to ensure the image scales properly across all screen sizes.

Reply:
Hey , Sorry for the late response, PFA other options.

1. **Option_1** : `flex flex-col` 
 
![image](https://github.com/user-attachments/assets/fd3e275b-e8d6-4c47-9de4-2fe7d11a1691)
2. . Option:_2: `size-3/12 sm:size-fit` 
 
![image](https://github.com/user-attachments/assets/8f7a1d42-79c5-4ff3-8a53-a544b43932f1)